andela/AndelaEatsApi
# #168325919 Add back button feature on slack bot

#### What does this PR do?
- Allow the moving back by a step to a previous interaction when using AndelaEats slack bot.

#### Description of Task to be completed?
-  addition of button frontend to the relevant file
- addition of button functionality

#### How should this be manually tested?
- Pull and checkout to this branch locally by running `git fetch` && `git checkout ft-add-slack-back-button-168325919`

- Run the local server and use a service like `ngrok` to create a URL slack would use.
- Create a slack bot in a slack-workspace you have admin rights to. It should have slash commands linked to `api/v1/bot` url.  URLS to be supplied in interactive components should be connected to `api/v1/bot/interactions/`
- If successful, interactions with the bot will have a back button added to them

- Run test `pytest -k pytest -k TestBotController`
- Run test `pytest -k pytest -k TestBotEndpoints`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#166788919](https://www.pivotaltracker.com/story/show/168325919)
